connectors/sources/jira.py

```diff
@@ -93,59 +92,75 @@ def get_default_configuration(cls):
         """
         return {
             "is_cloud": {
-                "value": True,
+                "display": "toggle",
                 "label": "True if Jira Cloud, False if Jira Server",
+                "order": 1,
                 "type": "bool",
+                "value": True,
             },
             "username": {
-                "value": "admin",
                 "label": "Jira Server username",
+                "order": 2,
                 "type": "str",
+                "value": "admin",
             },
             "password": {
-                "value": "changeme",
                 "label": "Jira Server password",
+                "sensitive": True,
+                "order": 3,
                 "type": "str",
+                "value": "changeme",
             },
             "service_account_id": {
-                "value": "me@example.com",
                 "label": "Jira Cloud service account id",
+                "order": 4,
                 "type": "str",
+                "value": "me@example.com",
             },
             "api_token": {
-                "value": "abc#123",
                 "label": "Jira Cloud API token",
+                "order": 5,
                 "type": "str",
+                "value": "abc#123",
             },
             "host_url": {
-                "value": "http://127.0.0.1:8080",
                 "label": "Jira host url",
+                "order": 6,
                 "type": "str",
+                "value": "http://127.0.0.1:8080",
             },
             "ssl_enabled": {
-                "value": False,
-                "label": "Enable SSL verification (true/false)",
+                "display": "toggle",
+                "label": "Enable SSL verification",
+                "order": 7,
                 "type": "bool",
+                "value": False,
             },
             "ssl_ca": {
```

Comment:
IMO, we should add `required=False` for this field since it is only required when `ssl_enabled` is set to `True`.

Reply:
This is handled by its dependency. If `ssl_enabled: True` then it should be required. If `False` then it won't be required/validated.